src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
+import { Person } from '../../types/Person';
+
+import { peopleFromServer } from '../../data/people';
+
+type Props = {
+  onSelected: (person: Person | null) => void;
+  delay: number;
+};
+
+export const Dropdown: React.FC<Props> = React.memo(({ onSelected, delay }) => {
+  const [value, setValue] = useState('');
+  const [appliedValue, setAppliedValue] = useState('');
+  const [focus, setFocus] = useState(false);
+
+  const applyValue = useCallback(debounce(setAppliedValue, delay), []);
```

Comment:
The same
<img width="1041" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/32290168-bdbe-4345-a855-8429494110ba">